client/controller/index.web.js

```diff
@@ -175,6 +180,18 @@ export function redirectLoggedOut( context, next ) {
 		loginParameters.redirectTo = 'https://wordpress.com' + loginParameters.redirectTo;
 	}
 
+	// If there is an issue with the auth cookie then we need to do more than
+	// just redirect to the login page. Otherwise that page may just detect that
+	// the user has already logged in. We fully log out to get fresh cookies.
```

Comment:
If you don't properly log out then you end up on this screen

<img width="2545" height="1460" alt="CleanShot 2025-10-30 at 18 41 01@2x" src="https://github.com/user-attachments/assets/c80cb1f7-bae9-4a45-841a-1a5bbd58fd30" />